tom_observations/utils.py

```diff
@@ -1,14 +1,79 @@
-from astropy.coordinates import get_sun, SkyCoord
-from astropy import units
+from astropy.coordinates import get_sun, SkyCoord, Angle, AltAz
+from astropy import units, coordinates
 from astropy.time import Time
 from astroplan import Observer, FixedTarget, time_grid_from_range
 import numpy as np
+from scipy import interpolate as interp
 import logging
 
 from tom_observations import facility
 
+#Work-around necessary until the NOAO main service upgrades are complete.
+#Would be sufficient to update astropy to v4 which handles this internally. But Wes doesn't want to do this to his work environment yet.
+#In future remove the two following lines and require an up to date astropy
```

Comment:
Comment lines should begin with # comment, ie space after the #. Again, may or may not be a issue depending on what, if any, format check they run on a pull request.

Reply:
Didn't realize they updated to v4. So I did locally, deleted that stuff, and checked functionality.